app/views/comment.html

```diff
@@ -1,4 +1,10 @@
-<div class="Comment-content">{{ comment.content }}</div>
+<div class="Comment-content">
+  {% if comment.title %}
+    {{ comment.title | escape }}
+  {% else %}
+    {{ comment.content | safe }}
```

Comment:
That means it's a question asked on another platform than BuddyUP so I'm not treating this use case. And this is what the app is doing before this patch too.